pkg/controller/cloudwatchmetrics/cloudwatchmetrics_controller.go

```diff
@@ -0,0 +1,111 @@
+package cloudwatchmetrics
+
+import (
+	"context"
+	"time"
+
+	integreatlyv1alpha1 "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var log = logf.Log.WithName("controller_cloudwatchmetrics")
+
+// Set the reconcile duration for this controller.
+// Currently it will be called once every 5 minutes
+const watchDuration = 5
```

Comment:
Can we replace this with seconds, just to allow for more accuracy

Reply:
Ok :+1: